### ISSUE TITLE
aur-chroot: rust error: linking with 'cc' failed: exit status: 1

### ISSUE THREAD

**Issue by echo-84** [AladW/aurutils#891]
When building rust packages in chroot the build fails with <code>error: linking with \`cc\` failed: exit status: 1</code>. Building without chroot and manually building with nspawn works. The particular package I am building is system76-power.
Same error across aurutils release, git, and master.

[system76-power-1.1.20-1-x86_64-build.log](https://github.com/AladW/aurutils/files/7814472/system76-power-1.1.20-1-x86_64-build.log)
[system76-power-1.1.20-1-x86_64-prepare.log](https://github.com/AladW/aurutils/files/7814473/system76-power-1.1.20-1-x86_64-prepare.log)
[aurutils.log](https://github.com/AladW/aurutils/files/7818571/aurutils.log)

**Comment by AladW** [comment]
> Manually building with nspawn

So `extra-x86_64-build` works?

The logs you provided don't tell much, I need a debug log as indicated in the issue template.

**Comment by echo-84** [comment]
Sorry about that, attached the compilation logs but forgot the aurutils logs. Updated the initial comment.

By 'manually with nspawn' I created a chroot with base-devel and then ran <code>systemd-nspawn -D /path/to/chroot</code> and built like normal.

I gave `extra-x86_64-build` a shot and it failed with the same error but building with [Classic Chroot](https://wiki.archlinux.org/title/DeveloperWiki:Building_in_a_clean_chroot#Building_in_the_chroot) worked.

I think it has something to do with the directory bindings changing something that breaks cargo's ability to link against libraries.
